debian/bitlbee-common.preinst

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+NAME=bitlbee
+PIDFILE=/var/run/$NAME.pid
+
+# git revision 7949d5a introduced dh_systemd.
+DH_SYSTEMD_REV="3.4.2+20160612+master+41-g7949d5a-git"
```

Comment:
Yeah, it's not used there, i just call this the "dh_systemd revision" because the packaging started using it, and the users probably have systemd units starting from that revision.